src/Npgsql/NpgsqlCommand.cs

```diff
@@ -109,6 +110,8 @@ public sealed partial class NpgsqlCommand : DbCommand, ICloneable
 
         internal const int DefaultTimeout = 30;
 
+        internal const bool DefaultIsPersistent = false;
```

Comment:
Not sure of the value of having a const false here... We can just say that the default is false without having a const to define it.